File: .codex/tmp_fix_index.js
const fs = require(''node:fs'');
const path = 'apps/cli/src/index.ts';
const text = fs.readFileSync(path, 'utf8');
const oldSnippet = 'void main();\n';
const newSnippet = `main().catch((error: unknown) => {
  const message = error instanceof Error ? error.message : String(error);
  const stack = error instanceof Error ? error.stack : undefined;
  process.stderr.write(`Fatal error: ${message}\n`);
  if (stack) {
    process.stderr.write(`${stack}\n`);
  }

  if (typeof process.exitCode !== 'number' || process.exitCode === 0) {
    process.exitCode = 1;
  }
});
`;
if (!text.includes(oldSnippet)) {
  throw new Error('pattern not found');
}
fs.writeFileSync(path, text.replace(oldSnippet, newSnippet), 'utf8');

File: .codex/tmp_fix_index.py
from pathlib import Path
import sys

path = Path(r"apps/cli/src/index.ts")
text = path.read_text(encoding='utf-8')
old = 'void main();\n'
new = """main().catch((error: unknown) => {
  const message = error instanceof Error ? error.message : String(error);
  const stack = error instanceof Error ? error.stack : undefined;
  process.stderr.write(`Fatal error: ${message}\\n`);
  if (stack) {
    process.stderr.write(`${stack}\\n`);
  }

  if (typeof process.exitCode !== 'number' || process.exitCode === 0) {
    process.exitCode = 1;
  }
});
"""
if old not in text:
    sys.exit('pattern not found')
path.write_text(text.replace(old, new), encoding='utf-8')
